packages/hydrogen/src/Numbers/UIAnimatedNumber.tsx

```diff
@@ -152,12 +152,9 @@ export function UIAnimatedNumber({
     const textLikeContainer = useTextLikeContainer();
 
     return (
-        <View
-            style={textLikeContainer}
-            testID={testID}
-            accessibilityLabel={`${formatted.value.integer}${formatted.value.decimal}`}
```

Comment:
Are you sure we should drop it? Maybe it make sense to leave it for accessibility?